README.md

```diff
@@ -0,0 +1,78 @@
+# Styled JSS
+
+## Usage
+
+### With Styled Creator
+
+Styled Creator allows to share classes between styled primitives
+
+```jsx
+import Styled from 'styled-jss'
+import injectSheet from 'react-jss'
+
+// Base styles, like a regular jss object.
+const styled = Styled({
```

Comment:
I think this should be a second separate example, because user will think he needs to do that.